test/e2e/geojson/geojson2mapml.html

```diff
@@ -0,0 +1,715 @@
+                             45.4049585
+                          ],
+                          [
+                             -75.6693647,
+                             45.3767494
+                          ],
+                          [
+                             -75.627064,
+                             45.3924229
+                          ],
+                          [
+                             -75.6467062,
+                             45.4215881
+                          ]
+                       ]
+                    ]
+                 },
+                 "properties":{
+                    "prop0":"This is a Polygon with holes"
+                 }
+              }
+           ]
+        };
+
+        let bbox_label_properties_and_caption_string = {
+           "title": "Point Geometry",
+           "bbox": [-75.916809, 45.886964, -75.516809, 45.26964],
+           "type": "FeatureCollection",
+           "features": [{
+               "type": "Feature",
+               //"bbox": [-75.916809, 45.886964, -75.516809, 45.26964],
+               "geometry": {
+                   "type": "Point",
+                   "coordinates": [-75.6916809, 45.4186964]
+               },
+               "properties": {
+                   "prop0": "This is a Point"
+               }
+           }]
+          };
+
+       let bbox_label_properties_and_caption_function = {
+           "title": "Point Geometry",
+           "bbox": [-75.916809, 45.886964, -75.516809, 45.26964],
+           "type": "FeatureCollection",
+           "features": [{
+               "type": "Feature",
+               //"bbox": [-75.916809, 45.886964, -75.516809, 45.26964],
+               "geometry": {
+                   "type": "Point",
+                   "coordinates": [-75.6916809, 45.4186964]
+               },
+               "properties": {
+                   "prop0": "This is a Point"
+               }
+           }]
+          };
+     
+          // --------------------------------------------------------------------------------- //
+          let mapml = document.getElementById('output'); 
+          let layer;
+
+          layer = geojson2mapml(point);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(line);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(polygon);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(multipoint);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(multilinestring);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(multipolygon);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(geometrycollection);
+          mapml.appendChild(layer);
+
+          layer = geojson2mapml(featurecollection);
+          mapml.appendChild(layer)
+
+          layer = geojson2mapml(bbox_label_properties_and_caption_string, {label: "Point", caption: "prop0", properties: "<h1>String Properties test</h1>"})

```

Comment:
Probably we need to have a few more options tests, but let's wait until we brainstorem about what other options might be useful for devs who use these functions.